ansible_navigator/actions/replay.py

```diff
@@ -5,24 +5,22 @@
 from .run import Action as BaseAction
 
 
-class MissingAttribute:
-    # pylint: disable=too-few-public-methods
-    """Raise an attribute error for any get"""
-
-    def __get__(self, instance, owner):
-        raise AttributeError()
-
-
 @actions.register
-class Action(MissingAttribute, BaseAction):
+class Action(BaseAction):
 
     # pylint: disable=too-many-instance-attributes
     """:replay"""
 
+    @property
+    def mode(self):
+        return self._args.mode
+
+    def run_stdout(self) -> int:
```

Comment:
since run for replay uses the run func in :run, can we use the run_stdout func in run too?  (just so interactive and stdout have the same pattern)?